proposals/stdlib/uri.md

```diff
@@ -0,0 +1,64 @@
+# Proposal: Add URI Type to Kotlin Standard Library
+
+**Author**: iseki zero  
+**Status**: Pending  
+**Submitted**: 2025-03-09  
+
+---
+
+## Summary
+
+This proposal advocates for introducing a first-class `URI` type to the Kotlin Standard Library. 
```

Comment:
It's against Kotlin naming convention, it must be called Uri (see for example recent UUID KEEP)

Reply:
Sounds great. I will fix it soon.